### PR TITLE
Added many minigames

### PR DESCRIPTION
**Pull request by Loup-Garou911XD**

### CODE REVIEW

File: plugins/minigames/invisible_one.py [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+class Team(ba.Team[Player]):
+    """Our team type for this game."""
+
+    def __init__(self, time_remaining: int) -> None:
+        self.time_remaining = time_remaining
+
+
+# ba_meta export game
+class InvicibleOneGame(ba.TeamGameActivity[Player, Team]):
```
Comment (rikkolovescats) [inline plugins/minigames/invisible_one.py:41]: this should be `InvisibleOneGame` i believe 
cc @itsre3

Comment (rikkolovescats) [reply plugins/minigames/invisible_one.py:41]: also lots of places in the source code

Comment (rikkolovescats) [reply plugins/minigames/invisible_one.py:41]: merging for now, let's make a version update for this change later

**Review by rikkolovescats — APPROVED** [review]
@Loup-Garou911XD thanks a ton for all the hard work that went into asking for consent from authors and compiling these minigames together! not to mention all the people who created these minigames in the first place, y'all are awesome!

**Comment by Loup-Garou911XD** [comment]
> @Loup-Garou911XD thanks a ton for all the hard work that went into asking for consent from authors and compiling these minigames together! not to mention all the people who created these minigames in the first place, y'all are awesome!

I am happy more PPL would enjoy these fun minigames

**Pull request merged**